libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -250,6 +265,10 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
                                 .ifBlank { "[no alternatives]" },
                         LOG_CATEGORY
                     )
+                    var printed = 0
+                    while (printed < refreshResponseJson.length) {
+                        printed += 4000
+                    }
```

Comment:
Good to remove?

Reply:
Argh, I removed the log but left this loop...